### DISCUSSION TITLE
Verify file (JSON) format and settings

### DISCUSSION THREAD

**Discussion by mwong-dotnet** [VerifyTests/Verify#1032] [Q&A]
Apologies I'm new to Verify so want to confirm the correct usage of the tool. 

I managed to get received.txt and verified.txt from:

`await Verifier.Verify(response.Content);`

The diff tool opens which is what I expected. However, the JSON was all on one line. 

JSON is better than text as I can view what is expected. received.json and verified.json files seem to work when I execute ` await Verifier.VerifyJson(response.Content);` Is this the correct use of VerifyJson?

What is the purpose and usage of  VerifierSettings.UseStrictJson();? 

To ignore certain members, should I be using `VerifierSettings.IgnoreMember`? If so, how is VerifierSettings different from creating an instance of VerifySettings and passing it as the 2nd  parameter in the above? 

Confused :(

Thanks in advance!

**Comment by SimonCropp** [comment]
can u supply a repro
